### PR TITLE
Fix unknown methods content stream (json on body in custom method set)

### LINKED ISSUE #92: Custom HTTP methods treated as GET
**RR version:** 1.3.1
**PHP:** 7.3

I noticed that custom methods (like `VOTE`) interpreted by rr like `GET` method and rr omits message body.

```php
$worker = new RoadRunner\Worker(new Goridge\StreamRelay(STDIN, STDOUT));
$client = new RoadRunner\PSR7Client($worker);

while ($request = $client->acceptRequest()) {
    $client->respond(new JsonResponse($request->getBody()->getContents()));
}
```

When I send a `VOTE` request, it returns me an empty body:
```bash
curl -X VOTE \
  http://localhost \
  -H 'Content-Type: application/json' \
  -d '{"test: "true"}'
```

And when I send `POST` request, it returns a correct body:
```bash
curl -X POST \
  http://localhost \
  -H 'Content-Type: application/json' \
  -d '{"test: "true"}'
```

How to make `rr` not to lose the message body?

### PR DESCRIPTION
**Pull request by Alex-Bond**
Resolves #92

### CODE REVIEW

File: service/http/request.go [hunk 1]
```diff
@@ -148,7 +148,7 @@ func (r *Request) Payload() (p *roadrunner.Payload, err error) {
 
 // contentType returns the payload content type.
 func (r *Request) contentType() int {
-	if r.Method != "POST" && r.Method != "PUT" && r.Method != "PATCH" && r.Method != "DELETE" {
+	if r.Method == "GET" && r.Method == "HEAD" {
```
Comment (acelot) [inline service/http/request.go]: `OPTION` method also does not contain the body

**Comment by wolfy-j** [comment]
Thank you.

**Comment by acelot** [comment]
@wolfy-j Can you draft a release?

**Comment by wolfy-j** [comment]
Yes, will do later today.

**Comment by wolfy-j** [comment]
There is another PR open https://github.com/spiral/roadrunner/pull/93 which i might want to solve first.

**Comment by wolfy-j** [comment]
@acelot patched in 1.3.2

**Pull request merged**